<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filter</class>
 <widget class="QWidget" name="Filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Filter</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="5">
    <widget class="QComboBox" name="comboBoxGeneratorLocation"/>
   </item>
   <item row="7" column="2">
    <widget class="TextBox" name="textBoxDelay">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>125</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Encounter</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QComboBox" name="comboBoxGeneratorLead"/>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="ComboBox" name="comboBoxGender">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>215</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>215</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Any</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">♂</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">♀</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="ComboBox" name="comboBoxGenderRatio">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>215</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>215</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Genderless</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">50% ♂ / 50% ♀</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">25% ♂ / 75% ♀</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">75% ♂ / 25% ♀</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">87.5% ♂ / 12.5% ♀</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">100% ♂</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">100% ♀</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLineEdit" name="TID">
     <property name="minimumSize">
      <size>
       <width>125</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>125</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>12345</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="labelLocation">
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Location</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="labelShiny">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Shiny</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QLineEdit" name="SID">
     <property name="minimumSize">
      <size>
       <width>125</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>125</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="placeholderText">
      <string>54321</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="checkBoxUseDelay">
     <property name="text">
      <string>Use Delay</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QComboBox" name="comboBoxGeneratorEncounter"/>
   </item>
   <item row="5" column="2">
    <widget class="CheckList" name="checkListNature">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>125</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="ComboBox" name="comboBoxShiny">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>125</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Any</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Star</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Square</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Star/Square</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="labelGame">
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Game</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="ComboBox" name="comboBoxGame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="maximumSize">
      <size>
       <width>215</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Ruby</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sapphire</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fire Red</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Leaf Green</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Emerald</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="labelGender">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Gender</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="CheckList" name="checkListHiddenPower">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>125</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="ComboBox" name="comboBoxAbility">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>215</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>215</width>
       <height>16777215</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Any</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QComboBox" name="comboBoxGeneratorPokemon"/>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="labelNature">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Nature</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="labelAbility">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Ability</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButtonEncounterSlotAny">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Any</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="labelTID">
     <property name="text">
      <string>TID</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="labelGenderRatio">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Gender Ratio</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="labelPokemon">
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Pokemon</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="labelEncounterSlot">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Encounter Slot</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QPushButton" name="pushButtonGeneratorLead">
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Cute Charm</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="labelHiddenPower">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Hidden Power</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="pushButtonHiddenPowerAny">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Any</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="pushButtonNatureAny">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Any</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="CheckList" name="checkListEncounterSlot">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>125</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QCheckBox" name="checkBoxDisableFilters">
     <property name="text">
      <string>Disable Filters</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="labelSID">
     <property name="text">
      <string>SID</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="11">
    <widget class="IVFilter" name="ivFilter" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/TextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>Forms/Controls/ComboBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>IVFilter</class>
   <extends>QWidget</extends>
   <header>Forms/Controls/IVFilter.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CheckList</class>
   <extends>QComboBox</extends>
   <header>Forms/Controls/CheckList.hpp</header>
   <slots>
    <slot>resetChecks()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxAbility</tabstop>
  <tabstop>comboBoxGender</tabstop>
  <tabstop>comboBoxGenderRatio</tabstop>
  <tabstop>checkListEncounterSlot</tabstop>
  <tabstop>pushButtonEncounterSlotAny</tabstop>
  <tabstop>checkListHiddenPower</tabstop>
  <tabstop>pushButtonHiddenPowerAny</tabstop>
  <tabstop>checkListNature</tabstop>
  <tabstop>pushButtonNatureAny</tabstop>
  <tabstop>comboBoxShiny</tabstop>
  <tabstop>checkBoxUseDelay</tabstop>
  <tabstop>textBoxDelay</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
